Explorer/Assets/Scripts/SceneRunner/SceneFacade.cs

```diff
@@ -105,8 +120,23 @@ public async UniTask StartUpdateLoopAsync(int targetFPS, CancellationToken ct)
 
                     stopWatch.Restart();
 
-                    // We can't guarantee that the thread is preserved between updates
-                    await runtimeInstance.UpdateScene(deltaTime);
+                    try
+                    {
+                        // We can't guarantee that the thread is preserved between updates
+                        await runtimeInstance.UpdateScene(deltaTime);
+                    }
+                    catch (OperationCanceledException) { throw; }
```

Comment:
This line is redundant: cancellation is caught by the outer catch

Reply:
Not quite redundant because the block contains: `catch (Exception e) { sceneExceptionsHandler.OnEngineException(e); break; }` which will also include `OperationCanceledException`